interfaces/builtin/udisks2.go

```diff
@@ -352,6 +353,15 @@ KERNEL=="sr*", ENV{ID_VENDOR}=="SanDisk", ENV{ID_MODEL}=="Cruzer", ENV{ID_FS_LAB
 ENV{ID_PART_TABLE_TYPE}=="dos", ENV{ID_PART_ENTRY_TYPE}=="0x0", ENV{ID_PART_ENTRY_NUMBER}=="1", ENV{ID_FS_TYPE}=="iso9660|udf", ENV{UDISKS_IGNORE}="0"
 `
 
+const udisks2PermanentSlotUDevTag = `
+# This file contains udev rules for UDisks2 service.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="sr*|md*|fd*|sd*|mmcblk[0-9]|mspblk[0-9]", TAG+="%[1]s"
+SUBSYSTEMS=="usb", TAG+="%[1]s"
```

Comment:
@jdstrand  I double-checked here
`fd[0-9]*` would be covered by `SUBSYSTEM=="block"`, do we really need to keep `fd[0-9]*` in the udev rule for udisk2 interface?